examples/globe_wfs_extruded.js

```diff
@@ -104,5 +106,41 @@ globeView.addLayer({
     },
 }, globeView.tileLayer);
 
+
+function setPointMaterial(result) {
+    var mesh = result.children[0];
+    mesh.material.size = 5;
+    mesh.material.sizeAttenuation = false;
+}
+
+function colorPoint(/* properties */) {
+    return new itowns.THREE.Color(0xff0000);
+}
+
+function selectRoad(properties) {
+    return properties.gestion === 'CEREMA';
+}
+
+globeView.addLayer({
+    type: 'geometry',
+    update: itowns.FeatureProcessing.update,
+    convert: itowns.Feature2Mesh.convert({
+        altitude: 400,
```

Comment:
We can do better: call `DEMUtils.placeObjectOnGround` (or `getElevationValueAt` if you want to place each point individually) in onMeshCreated instead.

Reply:
DEMUtils.placeObjectOnGround seem to place the object under the ground, for this example.
And DEMUtils.getElevationValueAt use coordinate and I don't have the coordinate at this point.

Your suggestion is good, but I'll do that in another PR. 

In this other PR, maybe I will add and option in the Feature2Mesh convertion to use DEMUtils.getElevationValueAt to place the feature at the good altitude during the conversion.

Reply:
> DEMUtils.placeObjectOnGround seem to place the object under the ground, for this example.

Maybe you could take a few minutes to investigate why this happen (it could be a bug and you could open an issue)?

Reply:
> DEMUtils.placeObjectOnGround seem to place the object under the ground, for this example.

Maybe your other PR could be a bugfix ? ;-) I'm actually not even kidding: they shouldn't be placed under the ground with `placeObjectOnGround`.

Furthermore:

> In this other PR, maybe I will add and option in the Feature2Mesh convertion to use DEMUtils.getElevationValueAt to place the feature at the good altitude during the conversion.

onMeshCreated gives you the mesh, that gives you the geometry, which in turns contains coordinates. I don't think you need another option. 

>  I'll do that in another PR.

I've already approved your PR, so I'm not requesting it here, though I really feel that these floating points are... odd and we should do something about it soon.

Reply:
Or we can rename it `placeObjectUnderGround`, good naming is important :rofl:

Reply:
Floating points is better than no point for now.
I'll look at that later.
Can you merge it ?

| Or we can rename it placeObjectUnderGround, good naming is important :rofl:
Lol !!!!! Thanks for that joke !!!